src/api/auth/auth.service.ts

```diff
@@ -84,8 +86,10 @@ export class AuthService {
     });
   }
 
-  async register(_dto: RegisterReqDto): Promise<RegisterResDto> {
-    throw new Error('Method not implemented.');
+  async register(dto: RegisterReqDto): Promise<RegisterResDto> {
+    this.mailService.sendEmailVerification(dto.email, 'test'); // TODO: Update logic when sending email verification
```

Comment:
💯 , tích hợp dùng worker luôn e hè, kiểu gì cũng phải có..

Reply:
@ngocp-0847 À đợi những PR chia ae cùng làm a ợ :v
PR này TO quá rồi